Sources/DeepLearning/Layer.swift

```diff
@@ -1169,6 +1169,34 @@ public struct UpSampling2D<Scalar: TensorFlowFloatingPoint>: Layer {
     }
 }
 
+/// An upsampling layer for 3-D inputs.
+@_fixed_layout
+public struct UpSampling3D<Scalar: TensorFlowFloatingPoint>: Layer {
+    @noDerivative public let size: Int
+
+    /// Creates an upsampling layer.
+    ///
+    /// - Parameter size: The upsampling factor for rows and columns.
+    public init(size: Int) {
+       self.size = size
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameter input: The input to the layer.
+    /// - Returns: The output.
+    @differentiable
+    public func call(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+        let shape = input.shape
+        let (batchSize, height, width, breadth, channels) = (shape[0], shape[1], shape[2]
+                                                            , shape[3], shape[4])
```

Comment:
When breaking a tuple to multiple lines, it's better to align elements that are not on the first line with the first element and not to make commas start a line, roughly:
```swift
(shape[0], shape[1], shape[2],
 shape[3], shape[4])
```

Another option is to put the entire tuple on the second line:

```swift
        let (batchSize, height, width, breadth, channels) =
            (shape[0], shape[1], shape[2], shape[3], shape[4])
```

What do you think?